### PR TITLE
Hi-fi: About- About Founder Section

### LINKED ISSUE #32: Lo-fi: About Page

### LINKED ISSUE #93: Hi-fi: Resource buttons

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This page contains the code for the **About Founder** section of the About page. 

**Other sections will be in other PRs for readability, in addition they are commented out of the about page to not skew the styling**

###TO DO: add modal for more resources

### CODE REVIEW

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Add object-fit: cover so there's no stretch 
image

**Review by jzhzhang66 — COMMENTED** [review]
For this part, since in the hi-fis part of the quote is italicized - would it be possible to call on renderRichText here so she can add italics where she wants and then we just display it in the typeface and font size? 

image

image

**Review by jzhzhang66 — COMMENTED** [review]
Everything else looks great!!

File: components/about-page/about-founder/AboutFounder.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  const isTabletOrMobile = useMatchMedia(
+    `(max-width: ${breakpointsObj.tabletLg}px)`,
+  );
+
+  const isMobile = useMatchMedia(`(max-width: ${breakpointsObj.tabletSm}px)`);
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const togglePopup = () => {
```
Comment (jean-zhang) [inline components/about-page/about-founder/AboutFounder.js]: nit: this can be one line

File: components/about-page/AboutPage.style.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    padding-bottom: 1.5em;
+  }
+  @media ${min.desktop} {
+    padding-bottom: 2em;
+  }
+`;
+
+const SquiggleUnderline = styled.img`
+  position: absolute;
+  width: clamp(9em, 44vw, 10em); //21vw; //clamp(10em, 95vw, 13em);
```
Comment (jean-zhang) [inline components/about-page/AboutPage.style.js]: is this intended to be part of the stuff you wanted commented out?

**Review by jean-zhang — APPROVED** [review]

**Comment by jean-zhang** [comment]
Seems like the donate/sign/etc buttons are a bit squished for the smaller sizes and have no padding
Screen Shot 2021-05-04 at 11 39 14 AM

**Comment by jean-zhang** [comment]
Also the modal doesn't seem to bring in the overlay/stop scrolling, but if that's intended for a later PR it's fine
Screen Shot 2021-05-04 at 11 40 44 AM

File: components/about-page/AboutPage.style.js (hunk 2 above)
Comment (isabellberzsenyi) [reply components/about-page/AboutPage.style.js]: lol totally didn't mean to commit that

**Review by jzhzhang66 — APPROVED** [review]
Yes looks so good!

**Pull request merged**